recipes/macholib/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "macholib" %}
+{% set version = "1.8" %}
+{% set sha256 = "323c9c8b85768244554b3c040808ed6393c783aa6eb1122e04dc8905f442e559" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - altgraph >=0.13
+
+test:
+  imports:
+    - macholib
```

Comment:
Could you also add a `commands` section that tests the `macho_find`, `macho_standalone`, and/or `macho_dump` scripts? These might require [adding `entry_point`s](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points) to the recipe to work.

Reply:
@dougalsutherland [Those entry points are deprecated (for the last 3 versions)](https://bitbucket.org/ronaldoussoren/macholib/src/121a955388293c67570d755598091bca5262fbb9/macholib/macho_dump.py?at=macholib-1.6.0&fileviewer=file-view-default#macho_dump.py-41) in favour of the code in `__main__.py`.

 I have added some `commands` tests with `python -m macholib find ...` as suggested in the deprecation message. I imagine one would need to run on some mac files to get a meaningful output though...

Reply:
Looting at CI, `python -m macholib standalone /tmp/` [doesn't exist with a status 0 in CircleCI](https://circleci.com/gh/conda-forge/staged-recipes/20821?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) because,
```
/tmp/ does not look like an app bundle
```
the `python -m macholib dump` also needs some specific file. The only one that can be run without those files is `python -m macholib find DIR`, I removed the other two.

Reply:
Do they take `--help` or something like that?

Reply:
I don't think so (it's not built with `argparse`),
```bash
% python -m macholib standalone --help || echo "FAILED"
/home/rth/.miniconda3/envs/test7-env/lib/python3.6/site-packages/macholib/__main__.py: --help does not look like an app bundle
FAILED
% python -m macholib standalone -h || echo "FAILED"
/home/rth/.miniconda3/envs/test7-env/lib/python3.6/site-packages/macholib/__main__.py: -h does not look like an app bundle
FAILED
% python -m macholib standalone || echo "FAILED" 
Usage:
  python -mmacholib dump FILE ...
  python -mmacholib find DIR ...
  python -mmacholib standalone DIR ...
FAILED
% python -m macholib --help || echo "FAILED"
Usage:
  python -mmacholib dump FILE ...
  python -mmacholib find DIR ...
  python -mmacholib standalone DIR ...
FAILED
```

Reply:
Referring to [this]( https://bitbucket.org/ronaldoussoren/macholib/src/53d85321baffbfd7d911572b5d37fbaa53056e5a/macholib/__main__.py?at=default&fileviewer=file-view-default#__main__.py-73:74 )? Yeah that doesn't look good. Not much we can do.

Besides one test already exercise that the file exists and can be called. So seems fine to me.

Reply:
FWIW raised this [issue]( https://bitbucket.org/ronaldoussoren/macholib/issues/23/add-help-option-with-clean-exit-status ) to get a normal `help` option.